crates/oxc_cli/src/command.rs

```diff
@@ -37,6 +37,11 @@ impl Command {
                     .help("File or Directory paths to scan. Directories are scanned recursively.")
                     .value_parser(ValueParser::path_buf()),
             )
+            .arg(
+              Arg::new("quiet")
```

Comment:
Im not sure if I got the command quite right, it only works if I run it like:
```
cargo run --bin oxc_cli lint foo.js -- --quiet
```
(note the `--` before `--quiet`)

If I run it as:
```
cargo run --bin oxc_cli lint foo.js --quiet
```

I get:
```
⚡ cargo run --bin oxc_cli lint foo.js --quiet
    Finished dev [unoptimized + debuginfo] target(s) in 0.31s
     Running `target/debug/oxc_cli lint foo.js --quiet`
error: unexpected argument '--quiet' found

  note: to pass '--quiet' as a value, use '-- --quiet'

Usage: oxc lint <PATH> [quiet]

For more information, try '--help'.
```

Reply:
What about `oxc_cli lint --quiet foo.js`?

Reply:
Looks like `.action(clap::ArgAction::SetTrue)` should do the trick, will finish up the PR and then push my changes :)